### PR TITLE
:bar_chart: ophi: Update Global MPI 2024

### PR DESCRIPTION
**Pull request by paarriagadap**
Data issue: https://github.com/owid/owid-issues/issues/1605

### CODE REVIEW

**Comment by paarriagadap** [comment]
Hi @Marigold! I was modifying the name of some variables and I found the error you see [here](https://buildkite.com/our-world-in-data/etl-automated-staging-environment/builds/11617#0192fc37-7599-44af-ae39-b6919311c6cb):
```
sqlalchemy.exc.IntegrityError: (pymysql.err.IntegrityError) (1062, "Duplicate entry 'grapher/ophi/2024-10-28/multidimensional_poverty_index/multidime' for key 'variables.idx_catalogPath'")
```

But I don't see I created repeated indicators, for example. Now I can't build the dataset and I get [this](https://buildkite.com/our-world-in-data/etl-automated-staging-environment/builds/11618#0192fc38-c761-4235-8028-bb0b6909a123):
```
sqlalchemy.exc.MultipleResultsFound: Multiple rows were found when one or none was required
```

I see the dataset [is now empty](http://staging-site-global-mpi-2024/admin/datasets/6779). Do you know why? Thanks!

**Comment by paarriagadap** [comment]
@Marigold Maybe this is similar to what you've found [here](https://github.com/owid/etl/pull/1701#issuecomment-1742713652)? I did see that before this failing the origins multiplied, and there where a lot (only two should show up)

**Comment by Marigold** [comment]
@paarriagadap sorry, it was caused by a recent change in MySQL. I fixed it in master and merged to this PR. It seems to be working.

**Comment by Marigold** [comment]
@paarriagadap sorry, it was caused by recent migration in MySQL. I merged with master and it should work now.

File: etl/steps/data/garden/ophi/2024-10-28/multidimensional_poverty_index.meta.yml [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+          - "{definitions.description_key_indicator}"
+          - "{definitions.description_key_flavor}"
+        presentation:
+          title_public: Share of the population deprived in the indicator <<indicator>>
+          title_variant: <<area>>, <<flavor>>
+        display:
+          name: Share of the population deprived in the indicator <<indicator>>
+          numDecimalPlaces: 1
+          <<: *common_display
+
```
[the remark this answers is not in the document]
Comment (paarriagadap) [reply etl/steps/data/garden/ophi/2024-10-28/multidimensional_poverty_index.meta.yml:304]: Thank you very much!

File: snapshots/ophi/2024-10-28/multidimensional_poverty_index_cme.csv.dvc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    title_snapshot: Global Multidimensional Poverty Index (MPI) - Current margin estimates (CME)
+    description_snapshot: |-
+      This dataset contains current margin estimates (CME), based on the most recent survey data.
+
+    # Citation
+    producer: Alkire, Kanagaratnam and Suppa
+    citation_full: |-
+      - Alkire, S., Kanagaratnam, U., and Suppa, N. (2024). The Global Multidimensional Poverty Index (MPI) 2024. Country Results and Methodological Note. OPHI MPI Methodological Note 58, Oxford Poverty and Human Development Initiative, University of Oxford.
+      - Alkire, S., Kanagaratnam, U., and Suppa, N. (2024). The Global Multidimensional Poverty Index (MPI) 2024. Disaggregation Results and Methodological Note. OPHI MPI Methodological Note 59, Oxford Poverty and Human Development Initiative, University of Oxford.
+    attribution: Alkire, Kanagaratnam and Suppa (2024) - The Global Multidimensional Poverty Index (MPI) 2024
```
[the remark this answers is not in the document]
Comment (paarriagadap) [reply snapshots/ophi/2024-10-28/multidimensional_poverty_index_cme.csv.dvc:28]: Yeah, as you guess it it comes mostly from the authors

**Pull request merged**